framework/test_framework.py

```diff
@@ -157,6 +159,31 @@ def deploy_test(platform):
         connection.connect_to_platform_port(diff_ports, test_config['log_echo'])
         terminate_children_processes(process)
 
+def move_results_to_output():

```

Comment:
Wouldn't it be better if this was done by default by the nix recipes? Seems like an unnecessary step done by the this tool, when the nix recipes could easily change the copy destination.

Reply:
Thanks for this point. I agree that implementing this change would enhance the cleanliness of the framework. However, it's important to note that the Nix recipes currently generate separate folders for each package. For instance, the output for the baremetal package goes to one folder, while the hypervisor package output is directed to a different one.

While it's possible to modify the output folder name within the package recipe, setting the path to the output directory seems to be a limitation. This is because Nix operates within a virtual environment, and as far as I know, changing the output directory path directly isn't feasible.